code/datums/ruins/space.dm

```diff
@@ -182,3 +182,71 @@
+	allow_duplicates = FALSE // This has its own area with an APC, and 2 APCs for the same area breaks badly
+
+/datum/map_template/ruin/space/blowntcommsat
+	id = "blowntcommsat"
+	suffix = "blowntcommsat.dmm"
+	name = "Blown-out Telecommunications Satellite"
+	description = "The remains of an old telecommunications satellite once utilised by NanoTrasen. It lays derelict, with quite a few pieces missing."
+	cost = 5 // This is a chonky boy
+	allow_duplicates = FALSE // This has its own area with an APC, and 2 APCs for the same area breaks badly. Its also huge.
+
+/datum/map_template/ruin/space/clownmime
+	id = "clownmime"
+	suffix = "clownmime.dmm"
+	name = "Clown & Mime Mineral Deposits"
+	description = "A crash site of two opposing factions, both trying to complete mining trips for their own valuable minerals. While all the crew have long perished, the minerals are likely intact."
+
+/datum/map_template/ruin/space/dj
+	id = "dj"
+	suffix = "dj.dmm"
+	name = "Russian DJ Station"
+	description = "An old russian listening station, long since defunct and lifeless, however the equipment is likely still in working condition."
+	cost = 2
+	allow_duplicates = FALSE // This has its own area with an APC, and 2 APCs for the same area breaks badly
+
+/datum/map_template/ruin/space/druglab
+	id = "drublab"
+	suffix = "druglab.dmm"
+	name = "Drug Lab"
+	description = "An old abandoned \"Chemistry\" site, which has a strong aura of amphetamines around it."
+	allow_duplicates = FALSE // This has its own area with an APC, and 2 APCs for the same area breaks badly
+
+/datum/map_template/ruin/space/syndiedepot
+	id = "syndiedepot"
+	suffix = "syndiedepot.dmm"
+	name = "Suspicious Supply Depot"
+	description = "A syndicate supply depot, heavily stocked, but heavily guarded with an assortment of shields, sentry bots, armed operatives and more."
+	allow_duplicates = FALSE // One of these is overpowered enough
+	always_place = TRUE // This is on the always spawn list because its already got a 50% chance to be shielded out, and may as well exist
+	cost = 5 // This is one of the biggest ruins on the server, so it has a high cost, not to mention the amount of loot you can get out of it
+
+/datum/map_template/ruin/space/ussp_tele
+	id = "ussp_tele"
+	suffix = "ussp_tele.dmm"
+	name = "USSP Teleporter"
+	description = "An old, almost fully destroyed teleporter, seemingly part of what used to be a much larger structure."
+	allow_duplicates = FALSE // This has its own area with an APC, and 2 APCs for the same area breaks badly
+
+/datum/map_template/ruin/space/ussp
+	id = "ussp"
+	suffix = "ussp.dmm"
+	name = "USSP"
+	description = "A decript station of seemingly russian origin. The last contact had with this station was a distress signal, and the rest was dark."
+	allow_duplicates = FALSE // One of these has enough loot
+	cost = 5 // This ruin is 100x100 tiles, so we dont want it to be treated like a 10x10 meteor
+
+/datum/map_template/ruin/space/whiteship
```

Comment:
Can you explain how someone is supposed to use this to find the other ruins?
When I spawned in and tested it, the console only listed "north of cyberiad" and "east of engiestation" as destinations.
In addition, when I clicked the "east of engiestation" destination on the whiteship console, it appeared to be broken.

Reply:
I will take a look at that, it worked for me

Reply:
![image](https://user-images.githubusercontent.com/25063394/84833071-67816300-b026-11ea-84c8-d1f5312508d0.png)
Fixed